JASP-Engine/JASP/R/anovarepeatedmeasures.R

```diff
@@ -995,68 +980,63 @@ AnovaRepeatedMeasures <- function(jaspResults, dataset = NULL, options) {
   confAdjust <- strsplit(confAdjust, " ")[[1]][1]
   
   if (!includeCI) {
-    correctionFootnote <- paste("P-value adjusted for comparing a family of", nEstimates)
+    correctionFootnote <- gettextf("P-value adjusted for comparing a family of %s", as.character(nEstimates))
   } else {
-    correctionFootnote <- paste("P-value and confidence intervals adjusted for comparing a family of", nEstimates,
-                                "estimates (confidence intervals corrected using the", confAdjust, "method).")
+    correctionFootnote <- gettextf("P-value and confidence intervals adjusted for comparing a family of %1$i estimates (confidence intervals corrected using the %2$s method).", nEstimates, confAdjust)
```

Comment:
`nEstimates` is a character int (I guess `regmatches()` returns a character because it's extracting a number from a string) and `confAdjust` is a string